nilearn/_utils/niimg_conversions.py

```diff
@@ -77,6 +78,10 @@ def _iter_check_niimg(niimgs, ensure_ndim=None, atleast_4d=False,
             if not _check_fov(niimg, ref_fov[0], ref_fov[1]):
                 if target_fov is not None:
                     from nilearn import image  # we avoid a circular import
+                    if target_fov == "first":
```

Comment:
Neither do I. Plus, it is false, we don't want the warning to be raised each time resampling on first image is asked: it may be asked but never applied.